kube/stateful_set.go

```diff
@@ -52,7 +52,7 @@ func NewStatefulSet(role *model.Role, settings *ExportSettings) (*v1beta1.Statef
 }
 
 // getVolumeClaims returns the list of persistent volume claims from a role
-func getVolumeClaims(role *model.Role) []v1.PersistentVolumeClaim {
+func getVolumeClaims(role *model.Role, createHealmChart bool) []v1.PersistentVolumeClaim {
```

Comment:
An even worse example is `headless bool, public bool`: https://github.com/SUSE/fissile/blob/master/kube/service.go#L42